WSL/connect-usb.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Connect USB devices to WSL 2 
+description: Learn how to connect a USB device to your WSL 2 Linux distribution using usbipd-win.
+ms.date: 11/09/2021
+ms.topic: article
+ms.localizationpriority: medium
+---
+
+# Connect USB devices to WSL 2
+
+This guide will walk through the steps necessary to connect a USB device to a Linux distribution running on WSL 2 using the USB/IP open-source project, [usbipd-win](https://github.com/dorssel/usbipd-win).
+
+Setting up the USB/IP project on your Windows machine will enable common developer USB scenarios like flashing an Arduino or accessing a smartcard reader.
+
+## Prerequisites
+
+- Running on either Windows 11 (Build 22000 or higher) or Windows 10 (Build ?? or higher).
+- Running [Linux kernel 5.10.60.1 or higher](./kernel-release-notes.md).
+
+> [!NOTE]
+> To check your Windows version and build number, select **Windows logo key + R**, type **winver**, select **OK**. You can update to the latest Windows version by selecting **Start** > **Settings** > **Windows Update** > **[Check for updates](ms-settings:windowsupdate)**.
+> To check your Linux kernel version, open your Linux distribution and enter the command: `uname -a`. To manually update to the latest kernel, open PowerShell and enter the command: 'wsl --update`.
+
+## Install the USBIP project
+
+Support for connecting USB devices is not natively available in WSL, so you will need to install the open-source usbipd-win project.
+
+1. Go to the [latest release page for the usbipd-win project](https://github.com/dorssel/usbipd-win/releases).
+2. Select the .msi file, which will download the installer. (You may get a warning asking you to confirm that you trust this download).
+3. Run the downloaded usbipd-win_x.msi installer file.
+
+> [!NOTE]
+> Alternatively, you can also install the usbipd-win project using [Windows Package Manager](/windows/package-manager/winget/) (winget). If you have already installed winget, just use the command: `winget install --interactive --exact dorssel.usbipd-win` to install usbipd-win. If you leave out --interactive, winget may immediately restart your computer if that is required to install the drivers.
+
+This will install:
+
+- A service called `usbipd` (display name: USBIP Device Host). You can check the status of this service using the Services app from Windows.
+- A command line tool `usbipd`. The location of this tool will be added to the PATH environment variable.
+- A firewall rule called `usbipd` to allow all local subnets to connect to the service. You can modify this firewall rule to fine tune access control.
+
+## Install the USBIP tools and hardware database
```

Comment:
```suggestion
## Install the USBIP tools and hardware database in Linux
```

Feel free to take or not take this suggestion, wondering if making it a bit clearer that these steps are done in WSL is valuable.